2021-04-16 10:05:52,476 INFO    MainThread:15427 [wandb_setup.py:_flush():69] setting env: {}
2021-04-16 10:05:52,477 INFO    MainThread:15427 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-16 10:05:52,477 INFO    MainThread:15427 [wandb_init.py:_log_setup():336] Logging user logs to /home/ubuntuadm/trainseg/utils/wandb/run-20210416_100552-24gpsi3g/logs/debug.log
2021-04-16 10:05:52,477 INFO    MainThread:15427 [wandb_init.py:_log_setup():337] Logging internal logs to /home/ubuntuadm/trainseg/utils/wandb/run-20210416_100552-24gpsi3g/logs/debug-internal.log
2021-04-16 10:05:52,477 INFO    MainThread:15427 [wandb_init.py:init():369] calling init triggers
2021-04-16 10:05:52,477 INFO    MainThread:15427 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-16 10:05:52,477 INFO    MainThread:15427 [wandb_init.py:init():418] starting backend
2021-04-16 10:05:52,478 INFO    MainThread:15427 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-16 10:05:52,507 INFO    MainThread:15427 [backend.py:ensure_launched():123] starting backend process...
2021-04-16 10:05:52,527 INFO    MainThread:15427 [backend.py:ensure_launched():127] started backend process with pid: 15449
2021-04-16 10:05:52,529 INFO    MainThread:15427 [wandb_init.py:init():423] backend started and connected
2021-04-16 10:05:52,531 INFO    MainThread:15427 [wandb_init.py:init():465] updated telemetry
2021-04-16 10:05:52,533 INFO    MainThread:15427 [wandb_init.py:init():484] communicating current version
2021-04-16 10:05:53,552 INFO    MainThread:15427 [wandb_init.py:init():489] got version response 
2021-04-16 10:05:53,553 INFO    MainThread:15427 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-16 10:05:53,787 INFO    MainThread:15427 [wandb_init.py:init():522] starting run threads in backend
2021-04-16 10:05:55,653 INFO    MainThread:15427 [wandb_run.py:_console_start():1524] atexit reg
2021-04-16 10:05:55,653 INFO    MainThread:15427 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-16 10:05:55,653 INFO    MainThread:15427 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-16 10:05:55,659 INFO    MainThread:15427 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-16 10:05:55,659 INFO    MainThread:15427 [wandb_init.py:init():546] run started, returning control to user process
2021-04-16 10:20:16,455 INFO    MainThread:15427 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 0
2021-04-16 10:20:16,456 INFO    MainThread:15427 [wandb_run.py:_restore():1466] restore
2021-04-16 10:20:18,716 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 877
  total_bytes: 877
}

2021-04-16 10:20:19,277 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 877
  total_bytes: 3693
}

2021-04-16 10:20:19,711 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3693
  total_bytes: 3693
}

2021-04-16 10:20:19,813 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3693
  total_bytes: 3693
}

2021-04-16 10:20:19,915 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3693
  total_bytes: 3693
}

2021-04-16 10:20:20,017 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3693
  total_bytes: 3693
}

2021-04-16 10:20:20,119 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3693
  total_bytes: 3693
}

2021-04-16 10:20:20,222 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3693
  total_bytes: 3693
}

2021-04-16 10:20:20,324 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3693
  total_bytes: 3693
}

2021-04-16 10:20:20,426 INFO    MainThread:15427 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3693
  total_bytes: 3693
}

2021-04-16 10:20:21,881 INFO    MainThread:15427 [wandb_run.py:_show_summary():1771] rendering summary
2021-04-16 10:20:21,882 INFO    MainThread:15427 [wandb_run.py:_show_history():1809] rendering history
2021-04-16 10:20:21,884 INFO    MainThread:15427 [wandb_run.py:_show_files():1838] logging synced files
